### DISCUSSION TITLE
标签页的Logo无法更改

### DISCUSSION THREAD

**Discussion by WYuHwn** [MaLuns/hexo-theme-async#121] [Q&A 疑难杂症] [answered]
当我在Favicon中指定了logo路径时，顶部logo可成功修改可是标签页logo无法修改。具体如图：
![image](https://cdn.jsdelivr.net/gh/WYuHwn/yam-images@main/question1.png)

**Comment by MaLuns** [comment, marked as answer]
![image](https://github.com/MaLuns/hexo-theme-async/assets/31614024/7cd66da2-7407-47c1-8bc5-676b21e5ca8a)

设置 icon32 icon16

**Comment by WYuHwn** [comment]
谢谢佬，已成功解决:)

WvHwn
________________________________
发件人: 白云苍狗 ***@***.***>
发送时间: Tuesday, May 28, 2024 4:54:39 PM
收件人: MaLuns/hexo-theme-async ***@***.***>
抄送: 山药 ***@***.***>; Author ***@***.***>
主题: Re: [MaLuns/hexo-theme-async] 标签页的Logo无法更改 (Discussion #121)


[Uploading image.png…]

―
